File: Euler10.java
public class Euler10 {
	public static void main (String[]args) {
		int n = 2000000;
		boolean [] primes = new boolean [n+1];
		for (int i = 0; i < primes.length; i++) {
			primes[i] = true;
		}
		
		for (int a = 2; a*a <= n; a++) {
			if (primes[a]) {
				for (int b = a*2; b <= n; b += a) {
					primes[b] = false;
				}
			}
		}
		long sum = 0;
		
		for (int j = 2; j < primes.length; j++) {
			if (primes[j]) {
				System.out.print(j + " + ");
				sum += j;
				
			}
		}
		System.out.println("\n" + sum);
	}
}

File: Python/Euler10.py
n = 2000000
primes = [False]*(n+1)
for i in range(0, len(primes)):
    primes[i] = True

a = 2
while((a*a)<=n):
    if(primes[a]):
        for b in range(a*2, n+1, a):
            primes[b] = False
    a = a + 1

sum = 0

for j in range(2, len(primes)):
    if(primes[j]):
        print("{} + ".format(j))
        sum = sum + j

print(sum)